Makefile

```diff
@@ -24,13 +24,12 @@ runserver: build_docker  ## Run server inside XBlock Workbench container
 
 .PHONY: clean
 clean:  ## Remove build artifacts
-	deactivate || true
-	rm -rf venv/
```

Comment:
This was from when `vagrant` created this automatically.
We shouldn't destroy a developer's virtualenv.